taca/flowcell_parser/classes.py

```diff
@@ -0,0 +1,322 @@
+
```

Comment:
It was supposed to be a separate repo, but francesco insisted that it became part of taca. I think it's a shame, because anyone might want to parse xten flowcells without requiring all of taca.